src/playcanvas-gltf.js

```diff
@@ -309,6 +309,11 @@ Object.assign(window, function () {
             if (resources.imagesLoaded === gltf.images.length) {
                 success();
             }
+
+            // Check if it's a blob DOMString so we can free it
+            if (image.src.startsWith('blob')) {
```

Comment:
Not keen on this to see if it is a blob string. Any better suggestions 😅 ?

Reply:
Is it possible for this URL to have been created by client/callback code? In that case, even if it is a blob you don't want to assume revoking is necessary. Perhaps you can store a flag when the URL is constructed indicating revoke is required?

Reply:
Good call, I was trying to be a bit too clever here. Using a flag instead